apisix/discovery/kubernetes/init.lua

```diff
@@ -298,88 +302,217 @@ local function create_endpoint_lrucache(endpoint_key, endpoint_port)
     return endpoint[endpoint_port]
 end
 
+
 local _M = {
     version = "0.0.1"
 }
 
-function _M.nodes(service_name)
-    local pattern = "^(.*):(.*)$"  -- namespace/name:port_name
-    local match = ngx.re.match(service_name, pattern, "jo")
-    if not match then
-        core.log.info("get unexpected upstream service_name:　", service_name)
-        return nil
-    end
 
-    local endpoint_key = match[1]
-    local endpoint_port = match[2]
-    local endpoint_version = endpoint_dict:get_stale(endpoint_key .. "#version")
-    if not endpoint_version then
-        core.log.info("get empty endpoint version from discovery DICT ", endpoint_key)
-        return nil
-    end
+local function start_fetch(handle)
+    local timer_runner
+    timer_runner = function(premature)
+        if premature then
+            return
+        end
 
-    return endpoint_lrucache(service_name, endpoint_version,
-            create_endpoint_lrucache, endpoint_key, endpoint_port)
+        local ok, status = pcall(handle.list_watch, handle, handle.apiserver)
+
+        local retry_interval = 0
+        if not ok then
+            core.log.error("list_watch failed, kind: ", handle.kind,
+                    ", reason: ", "RuntimeException", ", message : ", status)
```

Comment:
`"RuntimeException"` does not look like APISIX style logs

Reply:
But i think is kubernetes style

Reply:
People view APISIX logs from here, and we don't use their style of logs for every new ecology we dock.

Reply:
I think it's best practice to keep things as they are

In the List-Watch process, there are many error messages that may appear in many Kubernetes responses, all of them are in this format (Reason and Message)

It's not necessary to change the raw error message content for "apisix style"